docs/tools/vdb_table/data/paradedb.json

```diff
@@ -0,0 +1,190 @@
+{
+  "name": "ParadeDB",
+  "links": {
+    "docs": "https://docs.paradedb.com",
+    "github": "https://github.com/paradedb/paradedb",
+    "website": "https://paradedb.com",
+    "vendor_discussion": "https://github.com/superlinked/VectorHub/discussions/401",
+    "poc_github": "https://github.com/philippemnoel",
+    "slug": "paradedb"
+  },
+  "oss": {
+    "support": "full",
+    "source_url": "https://github.com/paradedb/paradedb",
+    "comment": ""
+  },
+  "license": {
+    "value": "AGPL-3.0 Licence https://opensource.org/license/agpl-v3",
+    "source_url": "https://opensource.org/license/agpl-v3",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "rust"
+    ],
+    "source_url": "https://github.com/paradedb/paradedb",
+    "comment": ""
+  },
+  "vector_launch_year": 2023,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://github.com/paradedb/paradedb",
+    "comment": "https://docs.paradedb.com/search/hybrid/basic"
+  },
+  "facets": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://github.com/orgs/paradedb/discussions/1110",
+    "comment": "PostGIS"
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.paradedb.com/search/dense/overview",
+    "comment": ""
```

Comment:
This docs page doesn't seem to describe the multi-vector search feature - does ParadeDB support storing and querying multiple vectors per document?

Reply:
I think this should be removed and marked as "none" - @dhruv-anand-aintech could you please confirm?

Reply:
yeah, I'll do that. @philippemnoel you can create a new PR with the appropriate docs url listed as source when you see this.